h2o-docs/src/product/data-science/glm.rst

```diff
@@ -563,7 +563,7 @@ The  negative log likelihood :math:`L(y_i,\mu_i)` function is:
 
 .. math::
 
- ^\text{max}_{\beta,\beta_0} \bigg[ \frac{-1}{N} \sum_{i=1}^{N}  \bigg \{ \bigg( \sum_{j=0}^{y_i-1} \text{log}(j + \theta^{-1} ) \bigg) - \text{log} (\Gamma (y_i + 1)) - (y_i + \theta^{-1}) \text{log} (1 + \alpha\mu_i) + y_i \text{log}(\mu_i) + y_i \text{log} (\theta) \bigg \} \bigg]
+   L(y_i,\mu_i) = \frac{-1}{N} \sum^{N-1}_{i=0} \big(\log \big(\Gamma (y_i + \theta^{-1}) \big) - \log \big(\Gamma (\theta^{-1}) \big) - \log \big(\Gamma (y_i + 1) \big) + y_i \log(\theta \mu_i) - (y_i + \theta - 1) \log (1 + \theta \mu_i) \big)
```

Comment:
Again, totally my fault:
![image](https://user-images.githubusercontent.com/7231712/208186634-35ce6adb-bca4-4ff1-aa12-2580de57655f.png)

The only thing to change is to replace the alpha with theta.